core/src/main/java/com/epam/esm/dao/provider/SortFactoryProviderImpl.java

```diff
@@ -12,9 +12,7 @@
 @NoArgsConstructor
 @Log4j2
 class SortFactoryProviderImpl implements SortFactoryProvider {
-
 	private static final Map<String, String> sortQueryFragments = new HashMap<>();
-
 	static {
```

Comment:
place static block before private static variable: code conventions
https://www.oracle.com/technetwork/java/codeconventions-150003.pdf
page 23

Reply:
It seems that compilator does not allow me to do this. As I put values (static block) into the static map (private static variable).